patterns/banner.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Banner
+ * Slug: woocommerce-blocks/banner
+ * Categories: WooCommerce
+ */
+?>
+<!-- wp:group {"align":"wide","style":{"color":{"background":"#f9eddb","text":"#443127"},"spacing":{"padding":{"right":"50px","bottom":"50px","left":"50px","top":"50px"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-text-color has-background" style="color:#443127;background-color:#f9eddb;padding-top:50px;padding-right:50px;padding-bottom:50px;padding-left:50px"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column {"style":{"spacing":{"padding":{"left":"110px"}}}} -->
+<div class="wp-block-column" style="padding-left:110px"><!-- wp:group {"style":{"spacing":{"padding":{"top":"60px","right":"0","left":"0"}}},"layout":{"type":"constrained","wideSize":"360px","justifyContent":"left"}} -->
+<div class="wp-block-group" style="padding-top:60px;padding-right:0;padding-left:0"><!-- wp:paragraph {"style":{"color":{"text":"#c85643"},"typography":{"fontSize":"18px"},"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}}} -->
+<p class="has-text-color" style="color:#c85643;margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;font-size:18px"><strong>HOLIDAY SALE</strong></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"48px"},"color":{"text":"#443127"},"spacing":{"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"padding":{"top":"0","right":"0","bottom":"0","left":"0"}}}} -->
+<p class="has-text-color" style="color:#443127;margin-top:0px;margin-right:0px;margin-bottom:0px;margin-left:0px;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;font-size:48px"><strong>Save up to 60%</strong></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"style":{"color":{"text":"#8e6d4e"},"typography":{"fontSize":"24px"},"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}}} -->
+<p class="has-text-color" style="color:#8e6d4e;margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;font-size:24px">Make the day special with our collection of discounted products.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons {"style":{"spacing":{"blockGap":"0","margin":{"top":"20px","bottom":"0"}}}} -->
+<div class="wp-block-buttons" style="margin-top:20px;margin-bottom:0"><!-- wp:button {"backgroundColor":"background","textColor":"foreground","style":{"typography":{"fontSize":"16px"},"border":{"width":"0px","style":"none"}},"className":"is-style-fill"} -->
+<div class="wp-block-button has-custom-font-size is-style-fill" style="font-size:16px"><a class="wp-block-button__link has-foreground-color has-background-background-color has-text-color has-background wp-element-button" style="border-style:none;border-width:0px">Shop Holiday Sales</a></div>
```

Comment:
Should we add the text & background color to the button?

Reply:
Hmmm I thought I did ![7eDjaY.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/1e4224f2-1b64-4caf-a9bf-1b42a58039be)

Are you not seeing that?

Reply:
Ok `I think` I know what's going on. If the theme happens to register a `default` color, even when you select that color (black/white) in the editor, it won't actually apply that CSS style property in code. It just adds the default color class name. This will cause issues in the design because different themes will apply different default colors. So this was confusing thinking I could just copy/paste the code into the patterns PHP after explicitly selecting the color. That is why you're not seeing the same thing on your end compared to me. I have pushed up changes to change default color class names with actual CSS properties.

Now I am thinking I have to make changes to all the other patterns I've already merged. doh!!

Reply:
Ahh yes, that has happened to me as well before, and Dan was saying it recently in a Slack conversation as well 😅 
Thanks for the changes! 😄